gustaf/helpers/init_helper.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any
+
+
+class LibraryCanNotBeLoadedHelper():
```

Comment:
Class name is very `C++`. Although I love it, we should maybe abbreviate. What do you think of `ModuleImportRaiser`.